tests/test_admin_and_files.py

```diff
@@ -41,6 +41,6 @@ def test_gitignore_has_correct_content(self):
 
 class IsStylesCSSExistTests(TestCase):
     def test_styles_exist(self):
-        file_exists = os.path.exists('static/css/styles.css')
+        file_exists = os.path.exists("static/css/styles.css")
 
         self.assertTrue(file_exists)
```

Comment:
Consider handling the case where the file doesn't exist. This could be done by asserting that an error is raised when the file does not exist.